### ISSUE TITLE
setupDockerCluster fails to run Vault container

### ISSUE THREAD

**Issue by fairclothjm** [hashicorp/vault-testing-stepwise#10]
`setupDockerCluster` broke after the vault docker image was deprecated in favor of the new Verified Publisher image hashicorp/vault. We tried fixing this in https://github.com/hashicorp/vault-testing-stepwise/pull/9 however more work is needed. The Vault container still fails to run.

**Comment by F21** [comment]
Any update for this? I am working on a Vault secrets plugin and want to implement a stepwise test, but the docker container doesn't start due to this bug.

I noticed the stepwise was removed from the terraform plugin: https://github.com/hashicorp/vault-plugin-secrets-terraform/pull/47

But, Vault still has 3 stepwise tests: https://github.com/search?q=repo%3Ahashicorp%2Fvault%20stepwise&type=code

Not sure if Vault is still running stepwise test or if they have some work around for the bug (tried to look through the GitHub actions log for Vault, but don't see any evidence of stepwise tests being run recently).
